### PR TITLE
docs(firestore): normalize region tags

### LINKED ISSUE #1817: firestore: normalizing sample region tags
Make sample's region tags consistent across all the languages.
[Inventorying and Normalizing Firestore Sample Region Tags](https://docs.google.com/document/d/1l4oECPCalM0-vCBDGxbvW-_UXzEU0efBsJ346CuKCkk/edit#heading=h.be87cnseibxs)
[Tags tracker](https://docs.google.com/spreadsheets/d/1aRCMW0krFoe2Gabg7_R3WSByoNXqSMox1nx5sdclMkQ/edit#gid=0)

### PR DESCRIPTION
**Pull request by AlisskaPie**
Closes #1817

### CODE REVIEW

**Comment by AlisskaPie** [comment]
I couldn't find a sample with this tag `fs_update_multiple_fields` on a site and there was no new region tag proposed in the tag tracker.
@tritone @BenWhitehead could you take a look?

File: firestore/firestore_quickstart/main.go [hunk 1]
```diff
@@ -15,7 +15,7 @@
 // Sample firestore_quickstart demonstrates how to connect to Firestore, and add and list documents.
 package main
 
-// [START fs_initialize]
+// [START firestore_setup_client_create]
```
Comment (BenWhitehead) [inline firestore/firestore_quickstart/main.go:18]: Thanks for opening this PR @AlisskaPie, we actually need to keep the existing tags and add the new tags alongside right now. At a later time we'll be able to come back and cleanup the existing tags.

Can you update the PR to look like the following:

```suggestion
// [START fs_initialize]
// [START firestore_setup_client_create]
```

**Review by BenWhitehead — CHANGES_REQUESTED** [review]
Can you update the PR so that all tags have their existing and new values similar to the suggestion in my other comment?

File: firestore/firestore_quickstart/main.go (hunk 1 above)
Comment (AlisskaPie) [reply firestore/firestore_quickstart/main.go:19]: Thank you for quick review, done!
Also, there're 5 samples that were not implemented on a site yet. I did them recently, its query watch samples (
fs_detach_listener, 
fs_listen_document,
fs_listen_errors,
fs_listen_changes,
fs_listen_multiple)
I guess we can safely rename these tags.

Comment (BenWhitehead) [reply firestore/firestore_quickstart/main.go:19]: I think this would be okay. @crwilcox Do you agree?

Comment (crwilcox) [reply firestore/firestore_quickstart/main.go:19]: I agree. Are these a match for a rows in the tracking spreadsheet? If not can we add rows? We are going to want to convert all `fs_name` snippets in the end.

Comment (AlisskaPie) [reply firestore/firestore_quickstart/main.go:19]: Yes, they are the same.
The one region tag left not replaced is `fs_update_multiple_fields` tag. There was no new region tag proposed for it in the tracking spreadsheet.

**Review by BenWhitehead — APPROVED** [review]

**Review by tritone — APPROVED** [review]
@tbpg The CI failures look like a test infra issue, can you investigate?

**Comment by tbpg** [comment]
Sent https://github.com/GoogleCloudPlatform/golang-samples/pull/1834 which will hopefully fix it.

**Comment by tmatsuo** [comment]
@tbpg 

snippet-bot is complaining because the snippet data still thinks `fs` is the right one.

This migration workflow is probably one of the edge cases that the current drift data can not cover.

I proposed that we have additional product prefix (in this case, `firestore` on top of `fs`), but they don't like the idea.

**Comment by tbpg** [comment]
Thanks, @tmatsuo! Makes sense.

**Review by tbpg — APPROVED** [review]

**Pull request merged**